src/types/pathways-model.d.ts

```diff
@@ -91,6 +91,12 @@ declare module 'pathways-model' {
     match: boolean; // in case expected !== actual but they are still a match
   }
 
+  export interface CriteriaResult {
```

Comment:
Don't forget we'll have to update the data model page on the wiki as well. Since the wiki isn't versioned alongside the code, could you post a comment or edit the description of this PR with the relevant piece of the wiki so we can just copy & paste when merging?

Reply:
Updated the description of the PR